docker-compose.yml

```diff
@@ -0,0 +1,28 @@
+services:
+
+  indexer:
+      build:
+        context: .
+        dockerfile: Dockerfile
+      ports:
+        - "8080:42069"
+      env_file:
+      - .env.docker
```

Comment:
Would using the [`.env.local` file](https://github.com/namehash/ensnode/blob/a2f3ce22dc993dc45db5444538bb5cf1310a5aff/.env.local.example) work here? It was meant to be loaded in to the indexer runtime. Let me know what you think 🙃